### ISSUE TITLE
Question regarding Whisper-finetune hyperparameters

### ISSUE THREAD

**Issue by NoLonGer15** [WingZLeung/TTDS#2]
Hi, thank you for releasing this repository and the TTDS work.

I am currently reproducing the Whisper-finetune part in this repo (forked from vasistalodagala/whisper-finetune), following the LOSO setting described in the paper.

I noticed that the README mentions that the filelists and configs for LOSO training will be added in a future update. At the moment, I am unsure whether my training setup matches the one used in the paper.

Could you please share the main training hyperparameters used for Whisper-finetune(learning rate, warm-up, epochs, and batch size)?

**Comment by WingZLeung** [comment]
Hi, thank you for your query.

We use the Bonet LOSO cross-validation:

val_speaker = 'TORGO_F03' if test_speaker != 'TORGO_F03' else 'TORGO_F04'

The optimal hyperparameters used for the Whisper large model were: learning rate=5e-06, warm up=500, batch size=32, epochs=20. The hyperparameters were optimised via grid search with the best checkpoint selected by the lowest validation WER.

**Comment by NoLonGer15** [comment]
Thanks for the clarification!
Could you also share the hyperparameters used for Whisper medium？
